spec/models/jobseeker_profile_spec.rb

```diff
@@ -22,6 +22,26 @@
         expect(profile.qualified_teacher_status_year).to eq(previous_application.qualified_teacher_status_year)
         expect(profile.qualified_teacher_status).to eq(previous_application.qualified_teacher_status)
         expect(profile.personal_details.first_name).to eq(previous_application.first_name)
+        expect(profile.personal_details.last_name).to eq(previous_application.last_name)
+      end
+
+      it "creates a job preferences record" do
+        expect(profile.job_preferences).to be_present
```

Comment:
sorry I've just noticed this. Is there a stronger assertion we can make?

Reply:
I added an assertion with a be_persisted matcher, not really sure what else to assert for an empty model